<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountPage</class>
 <widget class="QWidget" name="AccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,5,3">
     <item>
      <widget class="QRadioButton" name="read_radioButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="write_radioButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>填寫</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectTable_comboBox">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>基本資料</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="14,3,3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveTable_pushButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>儲存表格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OutputTable_pushButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>輸出表格</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="showTable">
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="insertRow_PushButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRow_PushButton">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>刪除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
